UPGRADE-6.3.md

```diff
@@ -209,8 +209,6 @@ monolog:
 ```
 
 Note that the `doctrine` channel MUST be excluded from database logging.
-Also note that the standard logging channel is not configured as "fingerscrossed" anymore. All messages for this channel
```

Comment:
Not sure if that is not still the case: there is also the deprecated standard logger (?)

Reply:
The old standard logger uses the "new" standard logger which is now configured as fingers-crossed (when doing it like suggested in chameleon-system/chameleon-system). So should be the same as before, or don't you think?